com.minres.coredsl/src/com/minres/coredsl/analysis/AnalysisContext.xtend

```diff
@@ -0,0 +1,20 @@
+package com.minres.coredsl.analysis
+
+import org.eclipse.xtext.validation.ValidationMessageAcceptor
+import java.util.Map
+import com.minres.coredsl.coreDsl.ISA
+import java.util.HashMap
+import java.util.Set
+
+class AnalysisContext {
+	
+	public val ValidationMessageAcceptor acceptor;
+	
+	/** Maps each ISA to a set of names declared within it and its parents */
+	public val Map<ISA, Set<String>> exposedDeclarations = new HashMap();
+	public val Map<ISA, ElaborationContext> elaborationResults = new HashMap();
```

Comment:
Prefer `LinkedHashMap` for its deterministic behaviour, unless we're absolutely sure we'll never have to iterate of these maps.

Reply:
This map is currently not used at all, but the ultimate goal is to return the AnalysisContext after the analyzer is done. So I agree that predictable behavior would be desirable in this instance. I will have to read up on the exact differences between those implementations.

Reply:
Switched to `LinkedHashMap`.